server/store.go

```diff
@@ -64,6 +64,10 @@ func (p *Plugin) fetchOAuthUserInfo(tokenKey, userID string) (*zoom.OAuthUserInf
 
 func (p *Plugin) disconnectOAuthUser(userID string) error {
 	encoded, err := p.API.KVGet(zoomUserByMMID + userID)
+	if len(encoded) == 0 {
+		return errors.Wrap(err, "Your Zoom account is not currently linked")
```

Comment:
```suggestion
		return errors.Wrap(err, "Your Zoom account is not linked with Mattermost yet, please try using `/jira connect`")
```

Reply:
A comment on top of Aaron suggestion: since we are on the zoom plugin, obviously he meant `/zoom connect` instead of `/jira connect` :smile:

Reply:
@larkox Do I need to use `errors.Wrap` or `errors.New` ?

Reply:
> Do I need to use `errors.Wrap` or `errors.New` ?

@yash2189 can we discuss this in the conversation here? https://github.com/mattermost/mattermost-plugin-zoom/pull/180#discussion_r513150886 I think my last comment there has enough context to make that decision